clusterman/aws/auto_scaling_resource_group.py

```diff
@@ -36,12 +35,10 @@
 logger = colorlog.getLogger(__name__)
 
 
-AutoScalingResourceGroupConfig = TypedDict(
-    'AutoScalingResourceGroupConfig',
-    {
-        'tag': str,
-    }
-)
+_BATCH_MODIFY_SIZE = 200
+CLUSTERMAN_STALE_TAG = 'clusterman:is_stale'
+
+logger = colorlog.getLogger(__name__)
```

Comment:
maybe github is just messing up its diff, but it looks like these already exist a couple lines up?

Reply:
Huh.  I wonder how that happened.